qiskit_nature/drivers/pyscfd/integrals.py

```diff
@@ -135,21 +135,21 @@ def _calculate_integrals(mol, hf_method='rhf', conv_tol=1e-9, max_cycle=50, init
     if isinstance(m_f.mo_coeff, tuple):
         mo_coeff = m_f.mo_coeff[0]
         mo_coeff_b = m_f.mo_coeff[1]
-        # mo_occ   = m_f.mo_occ[0]
-        # mo_occ_b = m_f.mo_occ[1]
+        mo_occ = m_f.mo_occ[0]
```

Comment:
If this can be inferred for now with the information in QMolecule (which is what we have done to date) can we do that to avoid refactoring the drivers to accommodate this now, and do it later along with whatever other driver refactoring is wanted. If we infer it in the QMolecule say, for now, this will then work with all the existing drivers without any need to go change them all.

Reply:
Rather than removing this working logic I am now handling (and testing) the case where `mo_occ is None` inside of the transformer.